scenarios/jwt_authentication/dotnet/jwt_authentication/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿using MQTTnet;
+using MQTTnet.Client;
+using MQTTnet.Client.Extensions;
+using System.Text;
+using Azure.Identity;
+using Azure.Core;
+
+//System.Diagnostics.Trace.Listeners.Add(new System.Diagnostics.ConsoleTraceListener());
+
+//MqttConnectionSettings cs = MqttConnectionSettings.CreateFromEnvVars();
+//Console.WriteLine($"Connecting to {cs}");
+
+// Create client
+IMqttClient mqttClient = new MqttFactory().CreateMqttClient(MqttNetTraceLogger.CreateTraceLogger());
+string hostname = "<Event Grid Mqtt Hostname Here>";
+
+// Create JWT
+var defaultCredential = new DefaultAzureCredential();
+var tokenRequestContext = new TokenRequestContext(new string[] { "https://eventgrid.azure.net/" });
+AccessToken jwt = defaultCredential.GetToken(tokenRequestContext);
```

Comment:
does this line trigger a UI to ask for user login?

Reply:
It will trigger a search for an existing login in this order: https://learn.microsoft.com/en-us/dotnet/api/azure.identity.defaultazurecredential?view=azure-dotnet

For me, having done the prerequisites for the other samples and having logged in via `az cli` it happens automatically.

Reply:
Are these steps documented above in the Readme? Is this a supported approach by EventGrid?